### ISSUE TITLE
lldpctl usage is inconsistent with actual args

### ISSUE THREAD

**Issue by baloo** [vincentbernat/lldpd#129]
lldpctl usage looks like the one of lldpcli
See:
```
# lldpctl -u /tmp/foo
lldpctl: invalid option -- 'u'
lldpctl: invalid option -- 'u'
Usage:   lldpctl [OPTIONS ...] [COMMAND ...]
Version: lldpd 0.7.15

-d          Enable more debugging information.
-u socket   Specify the Unix-domain socket used for communication with lldpd(8).
-f format   Choose output format (plain, keyvalue, xml, json).

see manual page lldpcli(8) for more information
```

**Comment by vincentbernat** [comment]
Thanks! Fixed in 5065264.

**Comment by baloo** [comment]
wow, so fast! thanks
